lib/surface/compiler/eex_engine.ex

```diff
@@ -318,42 +288,29 @@ defmodule Surface.Compiler.EExEngine do
             unquote(slot_props),
             unquote(slot_meta),
             unquote(module),
-            unquote(node_alias)
+            unquote(meta.node_alias)
           ),
           unquote(do_block)
         )
       end
-    end
-  else
-    # TODO: Remove when support for phoenix_live_view <= 0.15.5 is dropped
-    defp live_component_ast(
-           module,
-           context_expr,
-           props_expr,
-           dynamic_props_expr,
-           slot_props,
-           slot_meta,
-           module,
-           node_alias,
-           do_block
-         ) do
+    else
```

Comment:
Probably a nitpick: I believe it would help to be more explicit in this `else`. Either change it for `if component_type == Surface.Component` or use a case statement.

Reply:
@miguel-s that conditional will probably get a third branch with the functional component case. I'll change that in the other PR. Thanks!